web-app/src/components/CreateUser/container.js

```diff
@@ -174,13 +54,15 @@ export default Wrapped =>
     render() {
       return (
         <Wrapped
-          formElementsArray={getFormElementsArray(this.state.form)}
-          formIsValid={this.state.formIsValid}
-          submitForm={this.handleFormSubmit}
-          formChanged={(event, id) => this.handleFormChange(event, id)}
           success={this.state.success}
           error={this.state.error}
           loading={this.state.loading}
+          formData={this.state.formData}
+          formIsValid={this.state.formIsValid}
+          formStatus={(name, value, formIsValid) =>
```

Comment:
i might be being stupid here - but can this not be `formStatus={this.handleFormStatus(name, value, formIsValid)}`?

Reply:
When I do that, it says name, value and formIsValid are undefined?

Reply:
The fat arrow is binding those so they have scope within the function.